spec.bs

```diff
@@ -232,7 +232,157 @@ to conduct an auction to select an advertisement to display to the user, they ca
 {{Window/navigator}}.{{Navigator/runAdAuction()}} function, providing an auction configuration that
 tells the browser how to conduct the auction and which on-device recorded interests are allowed to
 bid in the auction for the chance to display their advertisement.
-  
+
+<h3 id="script-runners">Script Runners</h3>
+
+<xmp class="idl">
+[Exposed=InterestGroupBiddingScriptRunner]
+interface InterestGroupBiddingScriptRunner {
+  [NewObject] Promise<undefined> createScriptRunner(USVString script);

```

Comment:
As Paul mentioned, we might not want these createScriptRunner methods. I think I'm just not clear about how we would say "create a script runner" in runAdAuction's method steps.

Reply:
Then let's remove this from now and add a TODO somewhere around runAdAuction(). We don't want to introduce this API if we know it is wrong.

Reply:
Removed. There were todos for creating bidding/scoring script runners already.